src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -101,6 +99,59 @@ private void Analyze(TOptions analyzeOptions, AggregatingLogger logger)
             }
         }
 
+        protected virtual void ValidateOptions(TContext context, TOptions analyzeOptions)
+        {
+            bool succeeded = true;
+
+            succeeded &= ValidateFile(context, analyzeOptions.OutputFilePath, shouldExist: null);
```

Comment:
`shouldExist` is `bool`, not `bool?`, so this must be `false`. A little surprised this compiled.